keyboards/keyten/kt60_mu/kt60_mu.c

```diff
@@ -0,0 +1,12 @@
+// Copyright 2024 QMK Collaborators (@qmk)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include "quantum.h"
+
+#define LED_INDICATOR_PIN B11
+
+void matrix_init_kb(void) {
+    gpio_set_pin_output(LED_INDICATOR_PIN);
+    gpio_write_pin_high(LED_INDICATOR_PIN);
+    matrix_init_user();
+}
```

Comment:
Whats the value add here? Its not toggled by anything.

Reply:
This is just needed to keep the LED on continuously

Reply:
Wouldn't it make sense to set it as caps lock or some such?